nextflow.config

```diff
@@ -4,7 +4,7 @@ params {
 }
 
 process {
-    withLabel: 'gpu' {
+    withLabel('gpu') {
```

Comment:
Are there any processes that are labelled GPU or need a GPU. If not suggest removing